ledger-core/insolar/payload/proto_vnvn.r0.proto

```diff
@@ -202,7 +202,7 @@ message VDelegatedCallResponse {
 
     Reference Callee = 20;
     Reference CallIncoming = 21;
-    CallDelegationToken DelegationSpec = 22;
+    CallDelegationToken ResponseDelegationSpec = 22;
```

Comment:
why?

Reply:
Because it has different meaning in comparison with other messages. Here, we send this token to node which ask it. In others messages we use it as proof that we are delegated to do some action. And also we need that VDelegatedCallResponse doesn't implement tokenHolder interface